src/Containers/Chart.tsx

```diff
@@ -159,13 +159,13 @@ const CustomTooltip: React.FC<CustomTooltipProps> = ({
     payload,
     ...props
 }): React.ReactElement => {
-    const formatValue = (value) => value.toFixed(2);
+    const getFormattedHoverValue = () => payload[0].value.toFixed(2);
 
     if (active) {
         return (
             <CustomTooltipBox active={active} payload={payload} {...props}>
                 <strong>
-                    $<Big>{`${formatValue(payload[0].value)}`}</Big>
+                    $<Big>{`${getFormattedHoverValue()}`}</Big>
```

Comment:
Replace this whole line. in the React Tree i should only see
Either
1) <Component />
2) {GetComponent}
That way you can read the entire tree much clearer

Reply:
I don't get it

Reply:
You need to make the tree readable

Reply:
Yes, I understand this. I do not understand what changes I have to make to the code ...